src/entities.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""Generates entities.h and entities.c using entities.json from https://html.spec.whatwg.org/entities.json
+
+Delete entities.json if you want to fetch the latest version.
+The current version has 1 and 2 codepoint entities.
+If the spec is updated to include 3 or more codepoints per entity, then this script along with
+gdImageStringFTEx and gd_Entity_To_Unicode (from gdft.c) will need to be modified.
+A warning is issued for any such entities found and only the first 2 codepoints are used.
+"""
+
+import json
+import os
+import os.path
+import re
+import urllib.request
+from pathlib import Path
+
+__author__ = "Sterling Pickens"
+__copyright__ = "Copyright 2021, The libgd Project"
+__credits__ = ["Sterling Pickens", "Mike Frysinger"]
+__license__ = "GPLv3 or later"
```

Comment:
this is OK if you want to keep it, but libgd's license isn't GPL.  it's more BSD/MIT-ish.  so normally here we'd say like "MIT; see the COPYING file".

but again, if you want to keep it GPLv3+, that's OK if we put a disclaimer like "the script is under the GPLv3, but the generated header is under libgd's license".

Reply:
So you'd prefer replacing that line with the following ?
__license__ = "MIT; see the COPYING file"

Should the .c/.h files also contain the same line ?

Reply:
if you're OK with that change, that would make everything easier